AFM: Docs for the April release (#29)
mulesoft/docs-functional-monitoring: modules/ROOT/pages/afm-monitoring-private-apis.adoc

Private APIs are APIs with endpoints that are accessible only within your network. API Functional Monitoring (AFM) lets you monitor private APIs if you have a subscription to Anypoint Virtual Private Cloud (Anypoint VPC). With Anypoint VPC, you can run workers in CloudHub in a virtual, private, and isolated network segment, rather than in a region in which resources are shared.

> A _private location_ is a worker that runs in a CloudHub environment that is associated with an instance of Anypoint VPC. AFM can create workers within Anypoint VPC that can run tests against APIs that are accessible only within the network that Anypoint VPC is configured for. The APIs must be deployed in CloudHub. You can create multiple private locations in a single instance of Anypoint VPC. You can run tests against a private API only from a private location. However, you can run tests against a public API either from a private location or a public location.

Reviewer: GLOBALLY spell out AFM. I will stop marking this.

Before you can run tests from a private location, you must create a private location, which involves specifying a name and an environment for the worker to run in. The environment must be associated with an instance of Anypoint VPC and must be the same environment that an API to be tested is deployed in. For example, you might create a private location that you name `myLocation` and set it to run in environment `myEnvironment`. When you choose to run a test in `myLocation`, AFM allocates a worker in `myEnvironment` in Anypoint Virtual Private Cloud.

The following diagrams depict examples of configurations that are supported for monitors, private locations, APIs, and environments.

> * First example configuration

Reviewer: Consider rewording these list items to that they convey some meaning. Currently, these mean nothing to the audience.

+
image::afm-monitor-private-1.png[]
+

> A monitor runs on a single schedule. The schedule is associated with a private location. The monitor is written to test one or more endpoints in private API 1 and in private API 2, though it could also test endpoints in a larger number of private APIs. The private location and the APIs are all in a single environment.

Reviewer: ```suggestion
A monitor runs on a single schedule that is associated with a private location. The monitor is written to test one or more endpoints in private API 1 and in private API 2, although it could also test endpoints in a larger number of private APIs. The private location and the APIs are all in a single environment.
```

* Second example configuration
+
image::afm-monitor-private-2.png[]
+

> A monitor runs on two separate schedules, and each is associated with the same private location. The monitor is written to test one or more endpoints in private API 1 and in private API 2, though it could also test endpoints in a larger number of private APIs. The private locations and both APIs are all in a single environment.

Reviewer: ```suggestion
A monitor runs on two separate schedules, and each is associated with the same private location. The monitor is written to test one or more endpoints in private API 1 and in private API 2, although it could also test endpoints in a larger number of private APIs. The private locations and both APIs are all in a single environment.
```

* Third example configuration
+
image::afm-monitor-private-3.png[]
+

> A monitor runs on two separate schedules, and each is associated with a different private location. The monitor is written to test one or more endpoints in private API 1 and in private API 2, though it could also test endpoints in a larger number of private APIs.

Reviewer: ```suggestion
A monitor runs on two separate schedules, and each is associated with a different private location. The monitor is written to test one or more endpoints in private API 1 and in private API 2, although it could also test endpoints in a larger number of private APIs.
```

* Fourth example configuration
+
image::afm-monitor-private-4.png[]
+

> A monitor runs on two separate schedules, and each is associated with a different private location. Each private location is in a separate environment. One environment could be a test environment, and the other environment could be a production environment. Private API 1 and Private API 2 are copies of the same API. The monitor is written to test one or more endpoints in private API 1 and the same endpoints in API 2, though it could also test endpoints in a larger number of private APIs.

Reviewer: ```suggestion
A monitor runs on two separate schedules, and each is associated with a different private location. Each private location is in a separate environment. One environment could be a test environment, and the other environment could be a production environment. Private API 1 and Private API 2 are copies of the same API. The monitor is written to test one or more endpoints in private API 1 and the same endpoints in API 2, although it could also test endpoints in a larger number of private APIs.
```

Reviewer: GLOBALLY replace "though" with "although."

* Fifth example configuration
+
image::afm-monitor-private-5.png[]
+

> A monitor runs on one schedule, though it could run on more than one. The schedule is associated with a private location. The monitor is written to test one or more endpoints in public API, which is available on the world-wide web, 1 and in private API 2, though it could also test endpoints in a larger number of APIs.

Reviewer: "in public API" = which public API, or did you mean just "a" public API?
Oh...I think you mean...:
```suggestion
A monitor runs on one schedule, although it could run on more than one. The schedule is associated with a private location. The monitor is written to test one or more endpoints in public API 1, which is available on the web, and in private API 2, although it could also test endpoints in a larger number of APIs.
```
Since we are providing several specific situations, could it be confusing to audiences (or simply not necessary) to mention the "although"s?
